### PR TITLE
Scripts

### PR DESCRIPTION
**Pull request by pavpanchekha**

### CODE REVIEW

File: book/scripts.md [hunk 1]
```diff
@@ -239,7 +240,7 @@ and then give it a `log` property. We can do that *in JavaScript*:
 
 [console-log]: https://developer.mozilla.org/en-US/docs/Web/API/console/log
 
-``` {.javascript .example}
+``` {.javascript}
```
Comment (pavpanchekha) [inline book/scripts.md:243]: This means that in the book they have to not give this one a gray background.

File: book/scripts.md [hunk 2]
```diff
@@ -95,7 +95,8 @@ On the web, JavaScript is found in `<script>` tags. Normally, a
 to a JavaScript file, much like with CSS files. A `<script>` tag could
 also contain JavaScript source code between the start and end tag, but
 we won't implement that.^[It's a challenge for parsing, since it's
-hard to avoid less-than and greater-than signs in JavaScript code.]
+hard to avoid less-than and greater-than signs in JavaScript code. See
```
Comment (chrishtr) [inline book/scripts.md:98]: Not critical IMO

File: book/scripts.md [hunk 3]
```diff
@@ -206,7 +207,7 @@ that the other uses, say on a page like this:
 
 Suppose `a.js` is "`var x = 2;`" and `b.js` is "`console.log(x + x)`";
 the variable `x` is set in `a.js` but used in `b.js`. In real web
-browsers, that's important, since one script might define library
+browsers, that's common, since one script might define library
```
Comment (chrishtr) [inline book/scripts.md:210]: Not critical IMO

File: book/scripts.md [hunk 4]
```diff
@@ -612,7 +613,7 @@ for (var i = 0; i < inputs.length; i++) {
 ```
 
 Ideally, though we'd update the character count every time the user
-types into an input box, but that requires running JavaScript on every
+types into an input box. That requires running JavaScript on every
```
Comment (chrishtr) [inline book/scripts.md:616]: Not critical IMO

File: book/scripts.md [hunk 5]
```diff
@@ -1203,7 +1204,7 @@ included.[^text-children]
 [children]: https://developer.mozilla.org/en-US/docs/Web/API/Element/children
 
 [^text-children]: The DOM method `childNodes` gives access to both elements and
-    text.
+    text nodes.
```
Comment (chrishtr) [inline book/scripts.md:1207]: Not critical IMO

File: book/scripts.md [hunk 6]
```diff
@@ -1258,8 +1259,8 @@ reflect the *current* attributes of the element; for example:
 ``` {.javascript .example} 
 element.innerHTML = '<span id=foo>Chris was here</span>';
 element.id = 'bar';
-// Prints "<span id=bar>Chris was here</span>":
 console.log(element.innerHTML);
+// Prints "<span id=bar>Chris was here</span>":
```
Comment (chrishtr) [inline book/scripts.md:1263]: Not critical IMO

Comment (chrishtr) [reply book/scripts.md:1263]: Seems just as good to me to put the comment first

Comment (pavpanchekha) [reply book/scripts.md:1263]: Yeah I think it’s not critical, but with comment before one might be confused if that’s output from the previous line (like, maybe a print internal to the browser). Confusion would resolve if you think about it but after is unambiguous.

**Pull request merged**